content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -867,6 +873,50 @@ is disabled, `.spec.completions` is immutable.
 Use cases for elastic Indexed Jobs include batch workloads which require 
 scaling an indexed Job, such as MPI, Horovord, Ray, and PyTorch training jobs.
 
+### Pod Replacement Policy
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only set `podReplacementPolicy` on Jobs if you enable the `JobPodReplacementPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+{{< /note >}}
+
+By default, the Job controller recreates pods as soon they are either failed or terminating (have a deletion timestamp).
+This means that, at a given time, the number of running Pods for the Jobs can be greater than `parallelism` or, if using Indexed Jobs, more than one running Pod per index, if some of the Pods are terminating.
```

Comment:
```suggestion
This means that, at a given time, when some of the Pods are terminating, the number of running Pods for the Jobs can be greater than `parallelism` or greater than the running Pod per index (if using Indexed Jobs).
```

Not sure if my suggestion is still accurate, but the original sentence is hard to read

Reply:
I think you were very close.  I added one part that makes it correct.